core/tests/protractor.conf.js

```diff
@@ -338,6 +339,27 @@ exports.config = {
       }));
     }
 
+      // Adding a video reporter. For more information see
+      // https://www.npmjs.com/package/protractor-video-reporter
+    var _ADD_VIDEO_REPORTER = true;
```

Comment:
What's the point of this constant? Why not just remove the `if` statement that uses it since the value is always `true`?

Reply:
removed